programs/pluto/ikev2_ts.c

```diff
@@ -649,26 +900,502 @@ static int score_address_range(const struct end *end,
 	return f;
 }
 
+/**
+ * seclabel_narrow_status: Indicates the outcome of a security label "narrowing."
```

Comment:
The draft mentions that security label should not be narrowed.

   The mechanism of narrowing of Traffic Selectors with
   TS_IPV4_ADDR_RANGE and TS_IPV6_ADDR_RANGE does not apply to
   TS_SECLABEL as the Security Label itself is not interpreted and
   cannot itself be narrowed

Reply:
You are correct. I used "narrow" because I couldn't think of a better word while implementing this logic; hence, why I used quotes around it.

Do you think something like "select" would be better, i.e. "Indicates the outcome of a security label selection."

If you have a better term, please let me know!